platform/pc/drv/ps2/kbd/kbd.c

```diff
@@ -291,6 +294,7 @@ void key_handler()
             kbd_info.is_shift = true;
             break;
        case KBD_CAPS_PRESS_ID:
+            kbd_info.is_caps = !kbd_info.is_caps;
```

Comment:
Yes, it works. (Host: Windows 10 64-bit running VirtualBox VM 5.1.12)

P.S. Your review just shows to me a few seconds ago. Seems like you forgot to send?